decisions-embedded-mode-example/src/main/java/com/ibm/bamoe/decisions/embedded/DecisionsEmbeddedModeExample.java

```diff
@@ -0,0 +1,77 @@
+package com.ibm.bamoe.decisions.embedded;
+
+import java.util.Arrays;
+
+import org.drools.commands.SetActiveAgendaGroup;
+import org.drools.core.event.DebugRuleRuntimeEventListener;
+import org.kie.api.KieServices;
+import org.kie.api.runtime.ExecutionResults;
+import org.kie.api.runtime.KieContainer;
+import org.kie.api.runtime.KieRuntimeFactory;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.StatelessKieSession;
+import org.kie.dmn.api.core.DMNContext;
+import org.kie.dmn.api.core.DMNDecisionResult;
+import org.kie.dmn.api.core.DMNModel;
+import org.kie.dmn.api.core.DMNResult;
+import org.kie.dmn.api.core.DMNRuntime;
+import org.kie.internal.command.CommandFactory;
+
+public class DecisionsEmbeddedModeExample {
+
+    public static void main(String[] args) {
+        KieServices kieServices = KieServices.Factory.get();
+        KieContainer kieContainer = kieServices.getKieClasspathContainer();
+
+        System.out.println("-----> Now we execute rules in the stateful session <-----");
+
+        KieSession kieSession = kieContainer.newKieSession();
+        kieSession.addEventListener(new DebugRuleRuntimeEventListener());
+
+        ExecutionResults executionResults = kieSession.execute(
+            CommandFactory.newBatchExecution(Arrays.asList(
+                CommandFactory.newInsert(new Applicant("#0001", 20), "applicant"),
+                CommandFactory.newInsert(new LoanApplication("#0001"), "application"),
+                new SetActiveAgendaGroup("applicationGroup"),
+                CommandFactory.newFireAllRules()
+            ))
+        );
+
+        System.out.println(executionResults.getResults().get("application"));
+
+        kieSession.dispose();
+
+        System.out.println("-----> Now we execute rules in the stateless session <-----");
```

Comment:
I'm confused, but IINW there is no relationship with DRL execution and DMN one, and I can't see any embedding: could you please clarify that ?